packages/react-aria-components/docs/Table.mdx

```diff
@@ -499,13 +499,15 @@ A `Column` can be targeted with the `.react-aria-Column` CSS selector, or by ove
 
 ### TableBody
 
-A `TableBody` can be targeted with the `.react-aria-TableBody` CSS selector, or by overriding with a custom `className`.
+A `TableBody` can be targeted with the `.react-aria-TableBody` CSS selector, or by overriding with a custom `className`. It supports the following states and render props:
```

Comment:
TableBody is only showing the state isEmpty, should we edit that sentence or are there missing props?

Reply:
That is the only render prop for TableBody. I think we can keep the text to be consistent with other sections.

Reply:
My comment was that we aren't being consistent.

Reply:
I think we can refine these in a follow-up if needed. Wanted to keep the wording changes minimal in here to make it easier to review.